### ISSUE TITLE
nice rat

### ISSUE THREAD

**Issue by Tr0llMode** [gerald0mc/Mint-LEAKED-0.1.1#13]
https://moneymod3.is-a-skidded.host/​​‍‍​‍​‍‍‍​​​​‍​‍​‍‍‍​​‍​​‍‍​​
signexploit has a grabber

**Comment by gerald0mc** [comment]
> https://moneymod3.is-a-skidded.host/​​‍‍​‍​‍‍‍​​​​‍​‍​‍‍‍​​‍​​‍‍​​
> signexploit has a grabber

Yeah for your basic data so scary maybe read what it grabs next time
